packages/kbn-test/src/jest/setup/after_env.integration.js

```diff
@@ -12,3 +12,9 @@
  * Set the default timeout for the integration test suite to 10 minutes
  */
 jest.setTimeout(10 * 60 * 1000);
+
+/**
+ * Removed in Jest 27/jsdom, used in some transitive dependencies
+ */
+global.setImmediate = require('core-js/stable/set-immediate');
+global.clearImmediate = require('core-js/stable/clear-immediate');
```

Comment:
Hmm, should we remove the `setImmediate` imports and use this instead? Why is this and the `setup_test.js` version required?